pysat/instruments/pysat_testing.py

```diff
@@ -251,69 +178,87 @@ def load(fnames, tag=None, sat_id=None, sim_multi_file_right=False,
         data['dummy1'] = mlt_int
     data['dummy2'] = long_int
     data['dummy3'] = mlt_int + long_int * 1000.0
-    data['dummy4'] = num_array
+    data['dummy4'] = uts
     data['string_dummy'] = ['test'] * len(data)
     data['unicode_dummy'] = [u'test'] * len(data)
     data['int8_dummy'] = np.ones(len(data), dtype=np.int8)
     data['int16_dummy'] = np.ones(len(data), dtype=np.int16)
     data['int32_dummy'] = np.ones(len(data), dtype=np.int32)
     data['int64_dummy'] = np.ones(len(data), dtype=np.int64)
 
-    index = pds.date_range(data_date,
-                           data_date+pds.DateOffset(seconds=num-1),
-                           freq='S')
     if malformed_index:
-        index = index[0:num].tolist()
+        index = index[:].tolist()
```

Comment:
the [:] isn't doing anything at this point. index.tolist() should be good.

Reply:
fixed